src/_includes/partials/footer.html

```diff
@@ -12,22 +12,26 @@
                         href='mailto:web3opensourcecommunity@gmail.com' rel='noreferrer' target='_blank' title='Contact us via mail'></a>
 
                     <a aria-hidden='true'
-                        class='text-white fa fa-instagram fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        class='text-white fab fa-instagram fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
                         href='https://www.instagram.com/web3community/' rel='noreferrer' target='_blank'
                         title='Check our content at Instagram'></a>
 
                     <a aria-hidden='true'
-                        class='text-white fa fa-twitter fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        class='text-white fab fa-twitter fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
                         href='https://twitter.com/web3community' rel='noreferrer' target='_blank' title='Follow us via Twitter'></a>
 
                     <a aria-hidden='true'
-                        class='text-white fa fa-linkedin fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        class='text-white fab fa-linkedin fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
                         href='https://www.linkedin.com/company/web3community' rel='noreferrer' target='_blank'
                         title='Connect with us at LinkedIn'></a>
 
                     <a aria-hidden='true'
-                        class='text-white fa fa-github fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        class='text-white fab fa-github fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
                         href='https://github.com/web3community' rel='noreferrer' target='_blank' title='Contribute at GitHub'></a>
+                    
+                    <a aria-hidden='true'
+                        class='text-white fab fa-discord fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        href='https://discord.gg/TSRwqx4K2v' rel='noreferrer' target='_blank' title='Join our discord'></a>
```

Comment:
```suggestion
                        href='https://discord.gg/TSRwqx4K2v' rel='noreferrer' target='_blank' title='Join our Discord community'></a>
```

Reply:
ok I'll change the logo and resolve this